include/grpc/impl/codegen/grpc_types.h

```diff
@@ -272,6 +272,14 @@ typedef struct {
     grpc_ssl_session_cache*). (use grpc_ssl_session_cache_arg_vtable() to fetch
     an appropriate pointer arg vtable) */
 #define GRPC_SSL_SESSION_CACHE_ARG "grpc.ssl_session_cache"
+/** The caller of the secure_channel_create functions may override the target
```

Comment:
I would add a warning here

This should only be used for testing or when a client uses IP address to connect to a service.

Reply:
done